src/components/ui/TimeTrackerHeader.tsx

```diff
@@ -105,6 +106,7 @@ export default function TimeTrackerHeader({ id, workspaceName }: TimeTrackerHead
             if (stopTimerAPI.fulfilled.match(result)) {
                 reset()
                 setSelectedProjectId(null)
+                setResetProjectTrigger(prev => !prev) // Toggle to trigger TrackerInput reset
```

Comment:
This is the exact logic in the `handleManualSuccess`, why not just call it here?